src/server.tsx

```diff
@@ -153,8 +169,14 @@ app.post(
 // for it to run when `next()` function is called in the route handler
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 app.use((error: Error, req: express.Request, res: express.Response, next: express.NextFunction) => {
-    console.log('Something went wrong: ', error.message);
-    res.status(500).send({ error: error.message });
+    switch (error.constructor) {
+        case ValidationError:
+            res.status(400).send({ error: error.message });
```

Comment:
> Maybe we could also have a console.log inside this case?

Hey yeah that's a good question. I was also wondering what the right thing to do here was. In the end I didn't keep it since I thought that we'd be more concerned with 500s (which is a sign we've broken something). But happy to add and we can always remove it makes our logs too noisy. At least to begin with it'd probably be good to have this visibility.

Reply:
I've updated this to log in both cases (ec2cff3). It has an interesting side effect which is to log output during a test run. In general I have a strong preference to keep test output really clean/minimal (just the test and outcome ideally). We could stub `console.log` (or perhaps this should be `console.error` here), but I was concerned that might be confusing if you're trying to debug a test with `console.log`. What do you reckon?